statistics/handle/handle.go

```diff
@@ -416,9 +416,13 @@ func (h *Handle) MergePartitionStats2GlobalStats(sc *stmtctx.StatementContext, i
 				numBuckets = int64(hg.Len())
 			}
 		}
-		globalStats.Hg[i], err = statistics.MergePartitionHist2GlobalHist(sc, allHg[i], popedTopN, numBuckets)
-		if err != nil {
-			return
+		if numBuckets != 0 {
+			globalStats.Hg[i], err = statistics.MergePartitionHist2GlobalHist(sc, allHg[i], popedTopN, numBuckets)
+			if err != nil {
+				return
+			}
+		} else {
+			globalStats.Hg[i] = allHg[i][0].Copy()
```

Comment:
Only use to avoid some bugs. It will be deleted until these bugs have been fixed.